doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,241 @@
+.. _plutus_tools_component_descriptions:
+
+Plutus tools in development 
+=====================================
+
+The Plutus tools are currently in development. Early iterations of the tools have undergone testing as part of our research into their performance and features. Based on test results and feedback, our team has entered a new cycle of development to address certain design aspects and to place a greater focus on selected features and capabilities. 
+
+Logical components
+----------------------------------------------------
+
+Plutus tools consists of libraries, executables and logical components within Haskell packages for external developers to use. 
+A single Haskell package may contain multiple logical components. 
+
+The tools are located in the `plutus-apps <https://github.com/input-output-hk/plutus-apps>`_ repository. 
+
+For each tool or logical component shown below, we have indicated its specific location within plutus-apps and provided a brief description. 
+
+.. figure:: ./plutus-tools-ecosystem.png
```

Comment:
I would put the source of the image somewhere. Maybe as a comment above this line for now?

By source, I mean the link to the diagram in the PlanUML webserver.

Reply:
Also, some components are missing a description. Is that okay for now?

Reply:
I added a reference to the source of the diagram as a link. 
I'm not sure how to add descriptions in the diagram, but once we sort that out, we can update it and post it to the page.